02-peer-protocol.md

```diff
@@ -2721,12 +3330,39 @@ The sending node:
   - if it has sent `commitment_signed` for an interactive transaction construction but
     it has not received `tx_signatures`:
     - MUST set `next_funding_txid` to the txid of that interactive transaction.
+    - if it has not received `commitment_signed` for that interactive transaction:
+      - MUST set `next_commitment_number` to the commitment number of the `commitment_signed` it sent.
   - otherwise:
     - MUST NOT set `next_funding_txid`.
+  - if `option_splice` was negotiated:
+    - MUST set `your_last_funding_locked` to the txid of the last `splice_locked` it received.
+    - if it never received `splice_locked` for any transaction, but it received `channel_ready`:
+      - MUST set `your_last_funding_locked` to the txid of the channel funding transaction.
+    - otherwise (it has never received `channel_ready` or `splice_locked`):
+      - MUST NOT set `your_last_funding_locked`.
+    - if a splice transaction reached acceptable depth while disconnected:
+      - MUST set `my_current_funding_locked` to the txid of the latest such transaction.
+      - MUST send `splice_locked` for that transaction after `channel_reestablish`.
```

Comment:
Does this requirement supersede an earlier requirement:

```
- MUST wait to receive the other node's `channel_reestablish`
  message before sending any other messages for that channel.
```

Or must we wait for the other node's `channel_reestablish` before sending `splice_locked`?